tests/integration/utils/mock-api-vitest.ts

```diff
@@ -90,7 +91,7 @@ export const startMockApi = ({ routes, silent }: MockApiOptions): Promise<MockAp
   app.all('*', function onRequest(req, res) {
     addRequest(requests, req)
     if (!silent) {
-      console.warn(`Route not found: (${req.method.toUpperCase()}) ${req.url}`)
+      console.warn(`Route not found: (${req.method!.toUpperCase()}) ${req.url}`)
```

Comment:
What's the deal here? If `req.method` is optional, let's use optional chaining. If the type is wrong, can we fix it in tinyhttp?

Reply:
the type is inherited from `node:http` `IncomingMessage`